src/components/App.js

```diff
@@ -1,44 +1,54 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
 /* eslint-disable react/jsx-boolean-value */
 import React from 'react';
 import './App.css';
-import { Route } from 'react-router-dom';
-import Hero from './hero/Hero';
-import Header from './header/Header';
-import Lead from './lead/Lead';
-import Requests from './requests/Requests';
-import DummyPage from './dummy-page/DummyPage';
-import ChooseUs from './choose-us/ChooseUs';
-import Projects from './projects/Projects';
-import Footer from './footer/Footer';
+import { Route, Switch } from 'react-router-dom';
+import Homepage from './homepage/Homepage';
+import ProfessionPage from './profession-page/ProfessionPage';
+import PageNotFound from './page-not-found/PageNotFound';
 import pageTitles from '../arrays/projects-page-titles'; // import titles to be used in profession page
-import {
-  webDevProjects, // default display data for webdev profession page
-  dataAnalysisProjects, // default display data for dataAnalysis profession page
-  dataScienceProjects, // default display data for dataScience profession page
-} from '../arrays/delegate-tasks';
 
 function App() {
+  const [professionPage, setProfessionPage] = React.useState(false);
```

Comment:
Could we change `professionPage` to something that indicated it's boolean type? Something like `isProfessionPageFocused` or something?

Reply:
sure thang!